docs/Documentation/Systems/Kestrel/filesystems.md

```diff
@@ -67,8 +67,25 @@ To request a /shared-projects directory, please contact [hpc-help@nrel.gov](mail
 5. What you would like to call the directory. For example, "/shared-projects/myproject-shared" or other similar descriptive name, ideally between about 4-15 characters in length. 
 6. A group name for the UNIX group ownership of the directory, the same or similar to the directory name provided in Step 5. 
 ```
-!!! info
-    If you currently have a shared project directory on Eagle that you need copied over to Kestrel, please contact [hpc-help@nrel.gov](mailto:HPC-Help@nrel.gov). 
+
+!!! info 
+    Shared project directories are not automatically synced between Eagle and Kestrel. 
+
+**/kfs2/pdatasets**
+
+The /pdatasets directory on Kestrel hosts widely used data sets.
```

Comment:
Suggest either remove the / to just refer to it as "pdatasets" or use the full pathname "/kfs2/pdatasets"